website/src/components/ImageMasker.vue

```diff
@@ -98,19 +98,19 @@ export default {
 			required: true
 		},
 		data: {
-			type: String, // TODO: switch type to object and use facial recognition data to add mask
-			requied: true
+			type: Object, // TODO: switch type to object and use facial recognition data to add mask
+			required: true
 		}
 	},
 	data() {
 		return {
 			context: null,
 			originalDrawn: false,
 			maskSizing: {
-				leftOffset: 0.3,
-				topOffset: 0.4,
-				width: 0.5,
-				height: 0.2
+				leftOffset: 1,
```

Comment:
These default values are useful for if no face is found, why are we deleting them?